authorization-server/test/db_accesstokens.js

```diff
@@ -1,52 +1,40 @@
-/*jslint node: true */
-/*global it */
-/*global describe */
 'use strict';
 
-var assert = require("assert");
-var config = require('../config');
-var dbTokens = require('../' + config.db.type);
+const assert   = require('assert');
+const config   = require('../config');
 
-describe('access token saving/deleting', function () {
+const dbTokens = require(`../${config.db.type}`); // eslint-disable-line
 
-  it('should remove all tokens', function (done) {
-    dbTokens.accessTokens.removeAll(function () {
-      done();
-    });
+describe('access token saving/deleting', () => {
+  it('should remove all tokens', (done) => {
+    dbTokens.accessTokens.removeAll(() => { done(); });
```

Comment:
I would remove the brackets since it's on one line and use implicit return as in:

```js
dbTokens.accessTokens.removeAll(() => done());
```